files/etc/init.d/pbr

```diff
@@ -2202,12 +2190,10 @@ stop_service() {
 	resolver 'store_hash'
 	resolver 'cleanup_all'
 	resolver 'compare_hash' && resolver 'restart'
-	if [ "$enabled" -ne '0' ]; then
```

Comment:
this just suppresses the output when the service is stopped after it's been disabled (ie: uninstall scenario).